test/integration_tflint_test.go

```diff
@@ -109,3 +109,51 @@ func TestTflintFindsNoIssuesWithValidCodeDifferentDownloadDir(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, found)
 }
+
+func TestExternalTflint(t *testing.T) {
+	out := new(bytes.Buffer)
```

Comment:
Just wondering if we can run these 3 tests in parallel
```go
t.Parallel()
```

Reply:
TFLint tests are executed sequentially since during tflint invocation changes the current directory and in parallel tests this will lead to race condition in which directory tflint runs

Reply:
Ah, thanks for the explanation!